spec/javascripts/plots2_spec.js

```diff
@@ -1,70 +1,60 @@
-//= require application
-//= require jasmine-jquery
-//= require comment_expand
-
 var editor;
 
 describe("Plots2", function() {
 
   beforeEach(function() {
 
-    // for phantomjs running
-    jasmine.getFixtures().fixturesPath="../../spec/javascripts/fixtures";
-
-    // for in-browser running... still doesn't work
-    //jasmine.getFixtures().fixturesPath = 'assets/fixtures';
-    preloadFixtures('index.html', 'unlike.html', 'comment_expand.html');
-
-    jasmine.Ajax.install();
+    fixture.preload('index.html', 'unlike.html', 'comment_expand.html');
 
   });
 
+  it("sends a like request when like button is clicked", function() {
 
-  afterEach(function(){
-
-    jasmine.Ajax.uninstall();
-
-  });
-
+    fixture.load('index.html');
 
-  it("sends a like request when like button is clicked", function() {
+    ajaxStub = sinon.stub($, 'ajax', function(object) {
 
-    loadFixtures('index.html');
+		var response;
+		if   (object.url == '/likes/node/1/create') response = '4';
+		else response = 'none';
 
-    ajaxSpy = spyOn($, "ajax").and.callFake(function(object) {
+		// check this if you have trouble faking a server response:
+		if (response == '4'){
+            console.log('Faked response to:', object.url);
+		}
+		else console.log('Failed to fake response to:', object.url);
 
-      if   (object.url == '/likes/node/1/create') response = "4";
-      else response = 'none';
+		var d = $.Deferred();
+        if(response == '4'){
+		    d.resolve(response);
+        }
+        else{
+		    d.reject(response);
+        }
 
-      // check this if you have trouble faking a server response:
-      if (response != 'none'){
-        console.log('Faked response to:', object.url);
-        console.log(response);
-      }
-      else console.log('Failed to fake response to:', object.url);
+        return d.promise();
 
-      // http://stackoverflow.com/questions/13148356/how-to-properly-unit-test-jquerys-ajax-promises-using-jasmine-and-or-sinon
-      var d = $.Deferred();
-      d.resolve(response);
-      d.reject(response);
-      return d.promise();
+	});
 
-    });
 
     $('#like-button-1').trigger('click');
```

Comment:
Hi @jywarren, some help required here. I was trying to make this action work but its not calling the ajax function but in jasmine it was calling the ajax functions.